root/artist/edit_form.tt

```diff
@@ -26,14 +26,14 @@
           </span>
           [% field_errors(r.form, 'area.name') %]
         [% END %]
-        [% React.embed(c, 'static/scripts/edit/components/FormRowTextList', {
+        [% React.embed(c, 'static/scripts/edit/components/FormRowTextListSimple', {
```

Comment:
Wouldn't exactly the same issue affect ISNI and IPI codes as well?

Reply:
Yes, but they don't use `Controller::Role::IdentifierSet`. They're edited as part of the entity, while ISRCs/ISWCs have their own edit types.